package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "chalk": "^2.3.2",
     "minimist": "^1.2.0",
-    "react-scripts": "2.1.1",
+    "react-scripts": "^3.x",
```

Comment:
I think this might get us into trouble given the extent to which they change things out in minor and patch versions. I'll try and pick up the v2 branch and get that locked in first so I can make the appropriate tags on this repo.